### PR TITLE
Main loop working properly

### PR DESCRIPTION
**Pull request by StefanMermans**
objects are drawn properly. 
 - added a lot of documentation
 - Sorta reworked the project structure
 - added a test "scene"

### CODE REVIEW

**Review by bartmachielsen — DISMISSED** [review]
EVERYTHING

File: ProftaakPeriode4/ProftaakPeriode4/Model.cpp [hunk 1]
```diff
@@ -14,16 +16,41 @@ void Model::update()
...
 }
+
+void Model::InitTestObjects()
+{
+	// Test GameObjects
+	// TODO: remove
+	_lastTime = 0;
+
+	GameObject * camera = new GameObject();
+	CameraComponent * cameraComponent = new CameraComponent(1280.0f, 720.0f, 0.1f, 30.0f, 90.0f);
```
Comment (bartmachielsen) [inline ProftaakPeriode4/ProftaakPeriode4/Model.cpp:40]: ugly as fuck

Comment (StefanMermans) [reply ProftaakPeriode4/ProftaakPeriode4/Model.cpp:40]: Since when are test values ugly?

**Review by bartmachielsen — APPROVED** [review]
OK

**Pull request merged**